src/Desktop/UIAutomation/EventHandlers/NotificationEventListener.cs

```diff
@@ -30,7 +30,7 @@ public override void Init()
             }
         }
 
-        public void HandleNotificationEvent(IUIAutomationElement sender, NotificationKind kind, NotificationProcessing process, string displayString, string activityId)
+        public void HandleNotificationEvent(IUIAutomationElement sender, NotificationKind NotificationKind, NotificationProcessing NotificationProcessing, string displayString, string activityId)
```

Comment:
It's unfortunate that `IUIAutomationNotificationEventHandler.HandleNotificationEvent` uses upper-case parameter names. Would it make sense to suppress the warning here and use `notificationKind` and `notificationProcess` as the parameter names? Just a thought...

Reply:
Now I'm confused, since the [docs for this interface show lowercase parameters](https://docs.microsoft.com/en-us/windows/win32/api/uiautomationclient/nf-uiautomationclient-iuiautomationnotificationeventhandler-handlenotificationevent). Given that, I like the suppressing idea.

Reply:
Done.